lib/scrabble.rb

```diff
@@ -1,7 +1,20 @@
+require 'pry'
 class Scrabble
 
   def score(word)
-    1
+    letters = word.upcase.split("")
+
+    total_score = 0
+    letters_score = []
+    letters.each do |letter|
+      letters_score << point_values.values_at(letter)
```

Comment:
`point_values` returns a hash, so you should be able to access the score for any letter using more traditional hash syntax: `point_values[letter]`.